python/ray/_private/metrics_agent.py

```diff
@@ -23,12 +25,16 @@
 from opencensus.tags import tag_value as tag_value_module
 
 import ray
-import ray._private.prometheus_exporter as prometheus_exporter
 from ray._private.gcs_utils import GcsClient
 from ray.core.generated.metrics_pb2 import Metric
 
 logger = logging.getLogger(__name__)
 
+# Env var key to decide worker timeout.
```

Comment:
should we move this to where we define all the envs?

Reply:
where's that?